README.md

```diff
@@ -466,6 +466,35 @@ To build an Amazon-like autocomplete menu, suggesting popular queries and for th
 }
 ```
```

Comment:
Could we update every example of the README/examples with XSS protection on? Since this should be the default.